sw/host/tests/rom/e2e_bootup_bad_rom_ext_signature/BUILD

```diff
@@ -18,5 +18,6 @@ rust_binary(
         "//third_party/rust/crates:regex",
         "//third_party/rust/crates:structopt",
         "//third_party/rust/crates:tempfile",
+        "//third_party/rust/crates:rand",
```

Comment:
Looks like `deps` was alphabetically sorted, could we move `rand` up?

Reply:
I'm impressed you spotted this by eye James.

To save you time Douglas, here is the diff from the failed CI lint:

```diff
./sw/host/tests/rom/e2e_bootup_bad_rom_ext_signature/BUILD:
--- ./sw/host/tests/rom/e2e_bootup_bad_rom_ext_signature/BUILD	2022-10-18 14:46:44.186545819 +0000
+++ /tmp/buildifier-tmp-20582733	2022-10-18 14:52:03.087334649 +0000
@@ -15,9 +15,9 @@
         "//sw/host/opentitanlib",
         "//third_party/rust/crates:anyhow",
         "//third_party/rust/crates:humantime",
+        "//third_party/rust/crates:rand",
         "//third_party/rust/crates:regex",
         "//third_party/rust/crates:structopt",
         "//third_party/rust/crates:tempfile",
-        "//third_party/rust/crates:rand",
     ],
 )
```

Also remember there are the new helper *quality* rules:

```sh
# check bazel files
bazel run //quality:buildifier_check
# fix the a file
bazel run //quality:buildifier_fix -- $file
```

Reply:
good catch! Fixed!